src/solvers/flattening/boolbv_bswap.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************\
+
+Module:
+
+Author: Michael Tautschnig
+
+\*******************************************************************/
+
+#include "boolbv.h"
+
+#include <util/config.h>
+#include <util/invariant.h>
+
+bvt boolbvt::convert_bswap(const exprt &expr)
+{
+  if(expr.id()!=ID_bswap || expr.operands().size()!=1)
+    throw "expected bswap with one operand";
+
+  std::size_t width=boolbv_width(expr.type());
+
+  // width must be multiple of bytes
+  if(width%config.ansi_c.char_width!=0)
```

Comment:
Now that operator spacing is allowed in the CBMC codebase, let's use it to improve readability :)